test/scripts/install-kind.sh

```diff
@@ -5,13 +5,26 @@ set -ex
 export GO111MODULE="on"
 
 pushd $GOPATH/src/k8s.io/kubernetes/
-sudo ln ./_output/local/go/bin/kubectl /usr/local/bin/kubectl
-sudo ln ./_output/local/go/bin/e2e.test /usr/local/bin/e2e.test
+
```

Comment:
nit - I normally like extra blank lines for readability, but since this is in a pushd/popd, I would consider removing the blank line here and before the popd. Your call, I wouldn't argue too much if you want to keep them.

Reply:
done